added uri for proxy and ffmpeg for transcoding

lanefu opened the pull request:
Hi this patch adds an optional paramater `ServeUri` where you can specify the full URI of the server rather than the automatically generated one.   I had a usecase where i was running yt2pod behind an SSL proxy and i I needed to be able to make adjustments.

This also installs ffmpeg.  youtube-dl was unable to transcode correctly until It was added.

frou commented:
Thanks Lane. I wonder whether this parameter might be more appropriately called something like _LinkProxy_ because it's a different level of configuration to what the existing ones named _Serve*_ are for (starting up the web server in `main.go`).

I don't know whether you use Go much, but I've run the standard code formatter on your branch. You could pull that change then add another commit.

lanefu commented:
Yeah I don't do much with go, other than checkout the office go container and try to cross compile things to arm to run on some SBCs.

I've pulled, rebased and renamed the var as suggested.

frou commented:
I've added a paragraph to the readme about it. Thanks for the feature - in hindsight it was definitely something that was missing!
